packages/editor/src/editor/Editor.tsx

```diff
@@ -55,6 +55,7 @@ const Editor: React.FC<EditorProps> = ({
   lang = languages?.[0].lang ?? 'default',
   pluginsWillChange,
   cellSpacing,
+  components,
```

Comment:
this should already be passed

Reply:
We extract "components" from the "rest", so we need to pass it explicitly.